src/main/resources/jenkins/security/plugins/ldap/Messages.properties

```diff
@@ -1,2 +1,50 @@
 FromUserRecordLDAPGroupMembershipStrategy.DisplayName=Parse user attribute for list of groups
-FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for groups containing user
\ No newline at end of file
+FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for groups containing user
+LDAPSecurityRealm.LoginHeader=Login
+LDAPSecurityRealm.AuthenticationSuccessful=Authentication: successful
+LDAPSecurityRealm.AuthenticationFailed=Authentication: failed for user "{0}"
+LDAPSecurityRealm.AuthenticationFailedEmptyPass=Authentication: failed for user "{0}" with empty password
+LDAPSecurityRealm.UserId=User ID: {0}
+LDAPSecurityRealm.UserDn=User Dn: {0}
+LDAPSecurityRealm.UserDisplayName=User Display Name: {0}
+LDAPSecurityRealm.UserEmail=User email: {0}
+LDAPSecurityRealm.GroupMembership=Group membership:
+LDAPSecurityRealm.LookupHeader=Lookup
+LDAPSecurityRealm.UserLookupSuccessful=User lookup: successful
+LDAPSecurityRealm.UserLookupFailed=User lookup: failed for user "{0}"
+LDAPSecurityRealm.UserLookupInconclusive=User lookup: user "{0}" may or may not exist.
+LDAPSecurityRealm.UserLookupDoesNotExist=User lookup: user "{0}" does not exist.
+LDAPSecurityRealm.UserLookupBadCredentials=User lookup: bad credentials for user lookup.
+LDAPSecurityRealm.GroupMembershipMatch=User groups consistent (login and lookup)
+LDAPSecurityRealm.GroupMembershipMismatch=User groups inconsistent (login versus lookup)
+LDAPSecurityRealm.GroupLookupSuccessful=Group lookup: successful ({0,choice,1#{0} group|1<{0,number,integer} groups})
+LDAPSecurityRealm.NoDisplayName=No display name specified for user!
+LDAPSecurityRealm.EmptyDisplayName=Retrieved display name was empty!
+LDAPSecurityRealm.CouldNotRetrieveDisplayName=Could not retrieve the display name attribute.
+LDAPSecurityRealm.NoEmailAddress=No email address specified for user!
+LDAPSecurityRealm.EmptyEmailAddress=Retrieved email address was empty!
+LDAPSecurityRealm.CouldNotRetrieveEmailAddress=Could not retrieve the email address attribute.
+LDAPSecurityRealm.IsAttributeNameCorrect=Is the attribute name "{0}" correct?
+LDAPSecurityRealm.AvailableAttributes=Available attributes are:
+LDAPSecurityRealm.UserLookupManagerDnRequired=Is a Manager DN and password required to lookup user details?
+LDAPSecurityRealm.UserLookupManagerDnCorrect=Is a Manager DN and password correct?
+LDAPSecurityRealm.UserLookupManagerDnPermissions=Does the Manager DN have permissions to perform user lookup?
+LDAPSecurityRealm.GroupLookupManagerDnPermissions=Does the Manager DN have permissions to perform group lookup?
+LDAPSecurityRealm.GroupLookupManagerDnRequired=Is a Manager DN and password required to lookup group details?
```

Comment:
Tricky: I am referring to "Manager DN and password" as a single object. This becomes akin to asking whether it should be "Is there a pair of shoes on your feet" or "Are there a pair of shoes on your feet"... where you are trying to decide if you are focusing on the fact that there is 1 pair or 2 shoes